e2e/cypress/integration/messaging/visual_verification_of_tooltips_on_top_nav_channel_icons_posts_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Messaging', () => {
+    let testTeam;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+
+            // # Set up test channel
+            cy.apiCreateChannel(testTeam.id, 'channel-test', 'Public channel with a long name');
+
+            // # Set up Demo plugin
+            cy.apiInstallPluginFromUrl('https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.8.0/com.mattermost.demo-plugin-0.8.0.tar.gz', true);
+            cy.visit('/admin_console/plugins/plugin_com.mattermost.demo-plugin');
+            cy.get('[data-testid="PluginSettings.PluginStates.com+mattermost+demo-plugin.Enabletrue"]').check().should('be', 'checked');
+            cy.get('#saveSetting').click({force: true});
+            cy.wait(TIMEOUTS.ONE_MIN);
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+    it('MM-T134 Visual verification of tooltips on top nav, channel icons, posts', () => {
+        // * Hover effect wraps around Date and New Messages lines
+        cy.get('#postListContent').find('.top').should('be.visible');
```

Comment:
Would appreciate a recommendation on how to properly assert this.

Reply:
See above comment on how to have date line/separator.

For new message line, easiest is to mark a post by sysadmin as unread. You'll need to update to master since we recently added `cy.uiClickPostDropdownMenu` command.
```javascript
        cy.getLastPostId().then((postId) => {
            cy.uiClickPostDropdownMenu(postId, 'Mark as Unread');
        });
```

Doing so, you'll see
![Screen Shot 2020-09-30 at 9 05 36 PM](https://user-images.githubusercontent.com/5334504/94688834-b4ccda80-0360-11eb-9d77-4f9ab4dec798.png)

Then, maybe try to check CSS class or properties of those lines/separators.